src/CosmosDB/CosmosDB/ChangeLog.md

```diff
@@ -19,6 +19,17 @@
 -->
 
 ## Upcoming Release
+* This release introduces the cmdlets for the features of Continuous Backup(Point in time restore):
+  - Introduced support for creating accounts with continuous mode backup policy.
+  - Introduced support for Point in time restore for accounts with continuous mode backup policy.
+  - Introduced support to update the backup interval and backup retention for accounts with periodic mode backup policy.
+  - Introduced support to list the restorable resources in a live database account.
+  - New-AzCosmosBAccount cmdlet is updated with new parameters: IsRestoreReqeust, RestoreSource, RestoreTimestampInUtc, DatabasesToRestore, BackupPolicy.
+  - Update-AzCosmosDBAccount cmdlet is updated with new parameters: BackupPolicy.
```

Comment:
```suggestion
  - Added new parameters IsRestoreReqeust, RestoreSource, RestoreTimestampInUtc and DatabasesToRestore,BackupPolicy in `New-AzCosmosBAccount`
  - Added BackupPolicy in `Update-AzCosmosDBAccount`
```
Suggest to shorten release log as possible due to the length limit for release note
I remembered IsRestoreReqeust is changed, please check it.

Reply:
Shortened the message.